process-create-data-repo-request

```diff
@@ -0,0 +1,316 @@
+EXISTS_MSG = "Requested repository {repo} already exists"
+ACK_MSG = (
+    "Request received. I will create the requested repository after one of the following approve "
+    'the issue: {approvers_list}. You can do this by writing "+1" in a '
+    "comment."
+)
+COMPLETE_MSG = "Repository created: {url}"
+
+# -------------------------------------------------------------------------------
+# Statuses
+# --------------------------------------------------------------------------------
+# This is to determine the status of the issue after reading the labels
+
+# The issue has just been created
+NEW_ISSUE = "NEW_ISSUE"
+# The issue has been received, but it needs approval to start the build
+PENDING_APPROVAL = "pending-approval"
+# The build has been queued in jenkins
+FULLY_SIGNED = "fully-signed"
+
+# -------------------------------------------------------------------------------
+# Functions
+# --------------------------------------------------------------------------------
+
+
+#
+# Searches in the comments if there is a comment made from  the given users  that
+# matches the given pattern. It returns a list with the matched comments.
+#
+def search_in_comments(comments, user_logins, pattern, first_line):
+    found_comments = []
+    requested_comment_bodies = [c.body for c in comments if c.user.login in user_logins]
+    for body in requested_comment_bodies:
+        examined_str = body
+        if first_line:
+            examined_str = str(
+                body.encode("ascii", "ignore").decode().split("\n")[0].strip("\n\t\r ")
+            )
+
+        if examined_str == pattern:
+            found_comments.append(body)
+            continue
+
+        if re.match(pattern, examined_str):
+            found_comments.append(body)
+
+    return found_comments
+
+
+#
+# Adds a label to the issue in github
+# if dry-run is selected it doesn't add the label and just prints it
+def add_label(issue, label):
+    if opts.dryRun:
+        print("Not adding label (dry-run):\n %s" % label)
+        return
+    print("Adding label:\n %s" % label)
+    issue.add_to_labels(label)
+
+
+#
+# posts a message to the issue in github
+# if dry-run is selected it doesn't post the message and just prints it
+# if you set checkIfRepeated to False, if will not check if the message has already been written.
+#
+def post_message(issue, msg, comments, checkIfRepeated=True):
+    if opts.dryRun:
+        print("Not posting message (dry-run):\n %s" % msg)
+        return
+
+    if checkIfRepeated and search_in_comments(comments, ["cmsbuild"], msg, False):
+        print("Message already in the thread: \n %s" % msg)
+        return
+
+    print("Posting message:\n %s" % msg)
+    issue.create_comment(msg)
+
+
+#
+# reads the comments and gets returns the status of the issue
+#
+def get_issue_status(issue):
+    global labels
+    labels = [label.name for label in issue.get_labels()]
+    print("Issue Label: ", labels)
+
+    if not labels:
```

Comment:
I think it should be `if NEW_ISSUE in labels` ( where `NEW_ISSUE` should be `pending-assignment`)